CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+4.72.0
+=======
+### Features
+- Added MacOs Broker support. See [Issue #5051](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5051)
+- Retry policies for ManagedIdentities are now per request instead of per HttpManager [Issue #5252](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5252)
```

Comment:
```suggestion
- Retry policies for Managed Identity is now per request instead of per HttpManager [Issue #5252](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5252)
```

Reply:
I would remove this compelely or add it to a "engineering improvements". There is not feature update. HttpManager is not a customer-facing API

Reply:
Oh I see, didn't realize it was internal only. I can remove.